foundation_cms/blocks/image_block.py

```diff
@@ -0,0 +1,22 @@
+from wagtail.blocks import StructBlock, CharBlock, ChoiceBlock
+from wagtail.images.blocks import ImageBlock
+
+class CustomImageBlock(StructBlock):
```

Comment:
```suggestion
from wagtail.blocks import CharBlock, ChoiceBlock
from wagtail.images.blocks import ImageBlock
from foundation_cms.base.models.base_block import BaseBlock

class CustomImageBlock(BaseBlock):
```
New pattern is to inherit from the `BaseBlock`. I have a PR out there do retroactively update mine but let's catch this one now!